### DISCUSSION TITLE
pathfinder help!!!!

### DISCUSSION THREAD

**Discussion by ghost** [PrismarineJS/mineflayer#3443] [Q&A]
Hello!! I wanted to write a bot that would run after a specified list of items and pick them up. But I didn’t succeed, the bot runs halfway then stands and turns its head, the movement is very broken

If possible for version 1.16.5 minimum

**Comment by OmegaM1** [comment]
> Hello!! I wanted to write a bot that would run after a specified list of items and pick them up. But I didn’t succeed, the bot runs halfway then stands and turns its head, the movement is very broken
> 
> If possible for version 1.16.5 minimum

Have you tried using a more stable framework for the bot's movement? Consider implementing error handling or try optimizing the code to improve its performance.

**Comment by ghost** [comment]
In despair, I even tried through ChatGPT, but even that didn’t help((

**Comment by TerminalCalamitas** [comment]
Is your code for the pathfinder different from the [pathfinder example](https://github.com/PrismarineJS/mineflayer/blob/master/examples/pathfinder/gps.js)? If you could post the code you're using it would be easier to help find a solution.

**Comment by ghost** [comment]
Ok

**Comment by ghost** [comment]
> Is your code for the pathfinder different from the [pathfinder example](https://github.com/PrismarineJS/mineflayer/blob/master/examples/pathfinder/gps.js)? If you could post the code you're using it would be easier to help find a solution.


here is the code, please help if you can

**Comment by OmegaM1** [comment]
> In despair, I even tried through ChatGPT, but even that didn’t help((

Never use chatgpt for developing lol, Chatgpt gives you codes that look real, but the structure of all of them is old and illogical.
I suggest you to search on google or check out Stack Overflow to find your coding problems, it will help you to learn and you won't depend on chatgpt.

**Reply by ghost** [reply]
Okay  thanks

**Reply by ghost** [reply]
Can you write an example code in which the bot endlessly walks behind a stone lying on the floor, it's just that I've been trying to do it for 3 days((

**Reply by ghost** [reply]
Pleeeeease
